src/tests/flask/routes/chef_routes_test.py

```diff
@@ -0,0 +1,39 @@
+from src.tests.flask.base_flask_setup_test import BaseFlaskSetupTest
+
+
+class ChefApiTest(BaseFlaskSetupTest):
```

Comment:
Rename to `ChefApiE2ETest` and the file as well `chef_api_e2e_test.py`.

Reply:
done.